packages/gestalt/src/ExpandableModule.css

```diff
@@ -0,0 +1,5 @@
+.expandablemodule {
+  background: rgba(0, 0, 0, 0);
+  box-shadow: 0 0 8px rgba(0, 0, 0, 0.1);
```

Comment:
@galaxy0101 could we extract this into `./Borders.css`? And call it `boxShadow`? We only want 1 box shadow in Gestalt.